packages/@adobe/spectrum-css-temp/components/popover/index.css

```diff
@@ -20,6 +20,11 @@ governing permissions and limitations under the License.
 
   --spectrum-popover-min-width: var(--spectrum-global-dimension-size-400);
   --spectrum-popover-min-height: var(--spectrum-global-dimension-size-400);
+
+  /* Popover tip positioning for left/right oriented popovers (-16px/-19px) */
+  --spectrum-popover-tip-left: calc(-6px - var(--spectrum-global-dimension-size-125));
```

Comment:
should this have a different name since it is used by both left and right? Ditto for `--spectrum-popover-tip-top` being used by both top and bottom.

Reply:
good point, I've renamed them to `--spectrum-popover-tip-horizontal-offset` and `--spectrum-popover-tip-vertical-offset` respectively